packages/amplify_datastore/android/src/test/kotlin/com/amazonaws/amplify/amplify_datastore/AmplifyModelSchemaTest.kt

```diff
@@ -0,0 +1,518 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
```

Comment:
Looks like you're missing a `package` declaration.

Reply:
Is that necessary as this is a test class?  Another kotlin test (AmplifyDataStorePluginTest.kt) also does not have a package declaration.

Reply:
Both Java and Kotlin appear to _permit_ source files to compiled without `package` declarations:

In [Kotlin](https://kotlinlang.org/spec/packages-and-imports.html#packages-and-imports):
> an absence of a package header in a file means it belongs to the special root package.

In [Java](https://docs.oracle.com/javase/tutorial/java/package/createpkgs.html):
> If you do not use a package statement, your type ends up in an unnamed package. Generally speaking, an unnamed package is only for small or temporary applications or when you are just beginning the development process. Otherwise, classes and interfaces belong in named packages.

However, the practice is so rare an unconventional that I had to look it up. I do not recommend omitting a `package` header.

Reply:
Sounds good, I've added package declaration